exporter/lokiexporter/exporter_test.go

```diff
@@ -276,7 +276,7 @@ func TestLogsToLokiRequestWithGroupingByTenant(t *testing.T) {
 
 			// actualPushRequest is populated within the test http server, we check it here as assertions are better done at the
 			// end of the test function
-			assert.Equal(t, len(actualPushRequestPerTenant), len(tC.expected))
+			assert.Len(t, tC.expected, len(actualPushRequestPerTenant))
```

Comment:
```suggestion
			assert.Len(t, actualPushRequestPerTenant, len(tC.expected))
```

Might matter if the length doesn't match, in which case it prints out actual IIANM